src/components/PeopleList.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+// import { usePeople } from '../store/PeopleContext';
+
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleList: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const selectedPerson = slug ? slug : '';
+
+  function getPerson(persons: Person[], personName: string) {
+    return persons.find(person => person.name === personName) || null;
+  }
+
+  return (
+    <>
+      <>
+        <table
+          data-cy="peopleTable"
+          className="table is-striped is-hoverable is-narrow is-fullwidth"
+        >
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Sex</th>
+              <th>Born</th>
+              <th>Died</th>
+              <th>Mother</th>
+              <th>Father</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {people.map((person, index) => (
+              <tr
+                key={index}
```

Comment:
Using the index as a key in a list can lead to performance issues and unexpected behavior if the list changes. It's better to use a unique identifier from the `person` object, such as `person.id` or `person.slug`, if available.